package.json

```diff
@@ -4,29 +4,29 @@
   "private": true,
   "dependencies": {
     "@creativebulma/bulma-tooltip": "^1.2.0",
-    "@types/express": "^4.16.1",
-    "@types/react-redux": "^7.0.3",
-    "@types/react-router-dom": "^4.3.2",
-    "@types/redux": "^3.6.0",
-    "@types/socket.io": "^2.1.2",
-    "@types/socket.io-client": "^1.4.32",
-    "@types/uuid": "^8.0.0",
-    "express": "^4.16.4",
-    "i18next": "^15.0.7",
-    "i18next-browser-languagedetector": "^3.0.1",
+    "@types/express": "^4.17.7",
+    "@types/react-redux": "^7.1.9",
+    "@types/react-router-dom": "^5.1.5",
+    "@types/socket.io": "^2.1.10",
+    "@types/socket.io-client": "^1.4.33",
+    "@types/uuid": "^8.0.1",
+    "express": "^4.17.1",
+    "fsevents": "^2.1.3",
```

Comment:
Where does this one come from? It's no bueno if you're not on MacOS:
```
npm ERR! code EBADPLATFORM
npm ERR! notsup Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm ERR! notsup Valid OS:    darwin
npm ERR! notsup Valid Arch:  any
npm ERR! notsup Actual OS:   linux
npm ERR! notsup Actual Arch: x64
```

Reply:
This should be fixed now. I had added `fsevents@2.1.3` to package.json to resolve an issue. Forgot to take it out.

You may have to remove your `node_modules` and `package-lock.json` and do a fresh `npm install` after setting node version to 14

```
rm -rf node_modules package-lock.json
npm install
```